spec/service-meshes.pb.go

```diff
@@ -0,0 +1,215 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
This can be removed then I guess @leecalcote ?

Reply:
That is automatically generated by `protoc` and will be included each time `make build` runs.

Reply:
I'm not sure that we need to (or really should) have the generated classes stored in this repo.

Reply:
No actually, this is not the generated one. It was the old file when the rename was not done.

Reply:
Ooooh.

Reply:
> I'm not sure that we need to (or really should) have the generated classes stored in this repo.

I second that.